build/dependencies.props

```diff
@@ -13,6 +13,7 @@
     <MicrosoftAspNetIdentityEntityFrameworkPackageVersion>2.2.1</MicrosoftAspNetIdentityEntityFrameworkPackageVersion>
     <MicrosoftAspNetWebApiClientPackageVersion>5.2.2</MicrosoftAspNetWebApiClientPackageVersion>
     <MicrosoftAzureKeyVaultPackageVersion>2.3.2</MicrosoftAzureKeyVaultPackageVersion>
+    <MicrosoftAzureServicesAppAuthenticationPackageVersion>1.1.0-preview</MicrosoftAzureServicesAppAuthenticationPackageVersion>
```

Comment:
Why is this now no longer an external dependency?

Reply:
Microsoft.Azure.Services.AppAuthentication is a new external dependency of a package shipping in 2.0.8 for the first time, Microsoft.AspNetCore.AzureKeyVault.HostingStartup.

cc @pakrym

Reply:
Ah I see